src/language/goLanguage.ts

```diff
@@ -0,0 +1,78 @@
+import { Language, TreeSitterSyntaxNode } from './types.js'
+
+export const goLanguage: Language = {
+  toParameterTypeName() {
+    throw new Error('Unsupported operation')
+    // switch (node.type) {
```

Comment:
Consider removing the commented code in the 'toParameterTypeName' function or provide a reason for keeping it as a comment. If the code is not necessary, it's better to remove it to keep the codebase clean. [important]